Bucketeer/Sources/Internal/Model/MetricsEventData.swift

```diff
@@ -5,37 +5,147 @@ enum MetricsEventData: Hashable {
     case getEvaluationSize(GetEvaluationSize)
     case timeoutError(TimeoutError)
     case networkError(NetworkError)
+    case badRequestError(BadRequestError)
+    case unauthorizedError(UnauthorizedError)
+    case forbiddenError(ForbiddenError)
+    case notFoundError(NotFoundError)
+    case clientClosedError(ClientClosedError)
+    case unavailableError(UnavailableError)
     case internalSdkError(InternalSdkError)
+    case internalServerError(InternalServerError)
+    case unknownError(UnknownError)
 
     struct GetEvaluationLatency: Codable, Hashable {
```

Comment:
Please also rename it to `LatencyMetricsEvent` to keep consistency.

Reply:
Renamed 186567df71bdb469c2aeb208a0156cfa19fe83ec